distribution/pom.xml

```diff
@@ -21,13 +21,17 @@
         <artifactId>alliance</artifactId>
         <version>0.1-SNAPSHOT</version>
     </parent>
-    <groupId>org.codice.alliance</groupId>
+    <groupId>org.codice.alliance.distribution</groupId>
```

Comment:
Not a big fan of the groupId duplicating the artifactId, was there an issue that forced you to add `.distribution` ?